CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [Doc] Document --persist-replace in API section (#539)
 * [Fix] Fixed CI issue by updating `invalid_connection_string_duckdb` in `test_magic.py` (#631)
 * [Fix] Refactored `ResultSet` to lazy loading (#470)
+* [Fix] Showed depcrecation warnings message for plot API ([#513](https://github.com/ploomber/jupysql/issues/513))
```

Comment:
Please change the github link to the issue number since this will happen automatically during the release process.

`[Fix] Showed depcrecation warnings message for plot API (#513)`

[Contributing guide](https://github.com/ploomber/contributing/blob/main/contributing/pr.md#changelog)

Reply:
Got it, update it now